pkg/apis/apps/validation/validation.go

```diff
@@ -69,6 +69,26 @@ func ValidatePodTemplateSpecForStatefulSet(template *api.PodTemplateSpec, select
 	return allErrs
 }
 
+func ValidatePersistentVolumeClaimDeletePolicy(policy apps.PersistentVolumeClaimDeletePolicyType, fldPath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	switch {
```

Comment:
You can simplify this with `switch policy`

Reply:
done